### DISCUSSION TITLE
Why is it called omni completion?

### DISCUSSION THREAD

**Discussion by LunarLambda** [vim/vim#14072] [Q&A]
Just curious. It amuses me and my friends because one of my online handles is "omni". Couldn't find anything about it in the documentation.

**Comment by jmdevin** [comment]
I'm not sure if you wanted a more specific answer, but if you're not aware already, the prefix "omni" just means "all". You can see it here: [(link)](https://en.wiktionary.org/wiki/omni-#English) along with a list of other words that start with it.

**Comment by igbanam** [comment]
Vim's got many things it can complete

- filenames
- tags
- keywords
- functions
- spellings

…to name a few. `:h compl-omni` is thing-agnostic
